ui/pages/confirmations/components/approve-content-card/approve-content-card.js

```diff
@@ -45,6 +46,11 @@ export default function ApproveContentCard({
 }) {
   const t = useContext(I18nContext);
 
+  const tokenData = parseStandardTokenTransactionData(data);
+  const functionName = tokenData?.name;
+  const capitalizedFnName =
+    functionName?.charAt(0).toUpperCase() + functionName?.slice(1);
```

Comment:
double-checking, `!isSetApproveForAll` → `t('functionApprove')` seemed to be the catch-all case. It's okay to map the method name directly and there is no need for this catch-all anymore correct?

Reply:
Yes, this card is only shown for certain transaction types, and we're getting the function name from the ABIs, which means all cases should be covered.